Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/SignInSignUp.kt

```diff
@@ -96,13 +92,13 @@ fun SignInSignUpScreen(
     }
 }
 
+@OptIn(ExperimentalMaterial3Api::class) // SmallTopAppBar is experimental in m3
```

Comment:
I think this comment should have `CenterAlignedTopAppBar` instead?

Reply:
Yeah, you're right. I was originally using SmallTopAppBar but then changed it to CenterAlignedTopAppBar and missed updating my own comment :)